todo_app/data/session_items.py

```diff
@@ -82,3 +82,12 @@ def save_item(item):
     session['items'] = updated_items
 
     return item
+
+def complete_item(id):
```

Comment:
It looks like you're no longer using this file, so we can get rid of it now we've moved to Trello